docs/nuxt.config.js

```diff
@@ -5,16 +5,43 @@ const marked = require('marked')
 
 const renderer = new marked.Renderer()
 
+const ANCHOR_LINK_HEADING_LEVELS = [2, 3, 4, 5]
```

Comment:
H1 tags still need to have an ID associated with them for proper display of results in the search, as well as the top-level link in the right hand side bar table of contents (which allows the user to go back to the top easily without having to scroll)